radix-engine/src/kernel/kernel.rs

```diff
@@ -177,13 +177,11 @@ where
                 .current_frame
                 .get_node_visibility(&node_id)
                 .can_be_invoked(*is_direct_access),
-            Actor::Function { blueprint: _, .. } | Actor::VirtualLazyLoad { blueprint: _, .. } => {
-                true
-                // TODO: enable when package dependencies are explicitly declared.
-                // self
-                // .current_frame
-                // .get_node_visibility(blueprint.package_address.as_node_id())
-                // .can_be_invoked(false),
+            Actor::Function { blueprint, .. } | Actor::VirtualLazyLoad { blueprint, .. } => {
+                // TODO: Josh comment: what's the purpose of this?
```

Comment:
I roughly remember this is to check if we have `invoke` visibility into the package.